cibyl/orchestrator.py

```diff
@@ -191,6 +156,46 @@ def setup_sources(self):
                         if source.enabled:
                             source.setup()
 
+    def load_features(self):
+        """Read user-requested features and setup the right argument to query
+        the information for them."""
+        features_loader = FeaturesLoader()
+        user_features = self.parser.ci_args.get('feature')
+        if not user_features:
+            return []
+        if user_features and not user_features.value:
+            # throw error in case cibyl is called with --features argument but
+            # without any specified feature
+            msg = "No feature specified. Choose one of the following "
+            msg += "features:"
+            msg += features_loader.get_string_all_features()
```

Comment:
Right now this is what I get:
```
(.cibyl_venv) ➜  cibyl git:(features) cibyl --feature bla 
No feature bla. Choose one of the following features:
Network:
 * HA - Highly available Openstack deployment with at least 2 controllers.
 * IPV4 - Openstack deployment using IPv4.
 * IPV6 - Openstack deployment using IPv6
```

1. I would put a newline before each category
2. Maybe each category and/or feature can be in blue (or any other color but not red as the whole message)
3. I didn't check but we need to make sure that if the class doesn't have docstring for some reason, then it still looks fine and we don't see something like "* some_feature -"